bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -17,6 +17,10 @@ Action_MarkerFilter_WithoutGPS_Tooltip = Marker Filter\n\
                                          \n\
```

Comment:
`&` should be removed in all languages

![image](https://user-images.githubusercontent.com/1283445/236384639-eaeab1bf-4ef7-4b2f-992a-065ca8708fdb.png)

Reply:
> `&` should be removed in all languages
> 
Not sure why it shows on the UI since it should be used for the mnemonics. I will investigate

Reply:
I wonder if it's the [Nebula's SwitchButton](https://wiki.eclipse.org/Nebula_SwitchButton) that doesn't support mnemonics ?
At any rate, I have removed the `&` from all the strings